backend/dal/Helpers/Extensions/PropertyExtensions.cs

```diff
@@ -307,6 +307,26 @@ public static String GetLatestWorkflowCode(this Entity.Property property)
             return null;
         }
 
+        /// <summary>
+        /// Get the latest project status associated to this property, using the workflow sort order.
+        /// </summary>
+        /// <param name="property"></param>
+        /// <returns>The status of the latest project associated to this property</returns>
+        public static String GetLatestProjectStatus(this Entity.Property property)
```

Comment:
just copied from the above function for getting the project workflow, just altered slightly to get the status instead

Reply:
Looks like you're aware that because a property can be in multiple projects it's a bit of a grab bag to know which status to display.  Looks like you choose the largest `SortOrder` status.  Which "might" be the project in the most completed project.

Reply:
technically I chose this and he copied it, but yeah, I didn't see another available identifier to be able to get the rough order.